platform/security/certificate/security-certificate-generator/src/test/java/org/codice/ddf/security/certificate/generator/CertificateSigningRequestTest.java

```diff
@@ -36,33 +53,72 @@
 
     CertificateSigningRequest csr;
 
+    private static KeyPair makeKeyPair() throws Exception {
```

Comment:
Again, I am using the codice formatter and it moves it to that location every time because it is static.